pages/api/upload/index.js

```diff
@@ -0,0 +1,34 @@
+import formidable from "formidable";
+import cloudinary from "cloudinary";
+
+cloudinary.config({
+  cloud_name: process.env.CLOUDINARY_CLOUD_NAME,
+  api_key: process.env.CLOUDINARY_API_KEY,
+  api_secret: process.env.CLOUDINARY_SECRET,
+});
+
+export const config = {
+  api: {
+    bodyParser: false,
+  },
+};
+
+export default async function handler(request, response) {
+  if (request.method !== "POST") {
+    return response.status(400).json({ message: "Method not allowed" });
```

Comment:
I would split this into two lines, since it confuses students on a regular basis. They think that the response needs to be returned in order to appear in the client.
```suggestion
    response.status(400).json({ message: "Method not allowed" });
    return 
```